model/src/commonMain/kotlin/JobConfigurations.kt

```diff
@@ -80,6 +80,12 @@ data class AnalyzerJobConfiguration(
      */
     val environmentConfig: EnvironmentConfig? = null,
 
+    /**
+     * A flag indicating whether the submodules of the repository should be downloaded during the download process.
+     * If set to `true`, submodules will be downloaded; if `false`, they will be ignored.
+     */
+    val recursiveCheckout: Boolean = false,
```

Comment:
This should default to `true` backward compatibility.

Reply:
Done.